style.css

```diff
@@ -1,80 +1,636 @@
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable declaration-block-no-shorthand-property-overrides */
+/* stylelint-disable color-hex-length */
+/* stylelint-disable color-hex-case */
 /* stylelint-disable font-family-no-missing-generic-family-keyword */
 @import url('https://fonts.googleapis.com/css?family=Poppins:400,700,900');
 
 html {
   width: 375px;
-  height: 718px;
+  height: 4410px;
+  background-color: #f7f6f6;
 }
 
-header {
-  margin-top: 30px;
-  margin-bottom: 20px;
-  padding-left: 5%;
+.box1 {
```

Comment:
- [ ] Kindly remember to use `display: grid` for the section of your work as stated in the project requirement. :+1: